content/5_Plat/CHT.mdx

```diff
@@ -40,6 +40,15 @@ export const problems = {
 			['DP', 'convex'],
 			''
 		),
+		new Problem(
+			'CEOI',
+			'2009 - Harbingers',
+			'https://oj.uz/problem/view/CEOI09_harbingers',
+			'Normal',
+			false,
+			['DP', 'convex'],
+			'CHT on a tree -- dfs on the tree and keep track of which lines you remove so you can "restore" the deque of lines when leaving a node'
```

Comment:
Also, the official editorial is https://infosb.net/ceoi2009/tasks/solutions.doc (Mostafa has pdf versions on his GitHub as well)